lib/searchForKm.js

```diff
@@ -0,0 +1,16 @@
+const { collection } = require("./database");
+
+async function searchForKm(distance) {
+  console.log(distance);
+  const runs = await collection("rundata")
+    // .find({ distance: { $lte: distance } })
+    .find({ distance: distance })
```

Comment:
```suggestion
    .find({ distance: `${distance}` })
```

That should fix the filter problem ...

Reply:
Unfortunately still not.